packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -26,18 +26,23 @@ import { LivewireContourAnnotation } from '../../types/ToolSpecificAnnotationTyp
 import {
   AnnotationCompletedEventDetail,
   AnnotationModifiedEventDetail,
+  AnnotationModifiedEventType,
 } from '../../types/EventTypes';
 
 import { LivewireScissors } from '../../utilities/livewire/LivewireScissors';
 import { LivewirePath } from '../../utilities/livewire/LiveWirePath';
 import { getViewportIdsWithToolToRender } from '../../utilities/viewportFilters';
 import ContourSegmentationBaseTool from '../base/ContourSegmentationBaseTool';
 
+const { isEqual } = csUtils;
+
 const CLICK_CLOSE_CURVE_SQR_DIST = 10 ** 2; // px
 
 class LivewireContourTool extends ContourSegmentationBaseTool {
   public static toolName: string;
   private scissors: LivewireScissors;
+  /** The scissors from the right handle, used for editing */
```

Comment:
what is right handle?

Reply:
Yeah... the name is a bit confusing. After reading the code I saw that `handle-1` is called `left` and `handle+1` is called `right`. I thought about `editScissors` but I'm still not happy with it.

Reply:
They are the left and right handles relative to a clockwise direction set of handles.  Could alternatively call them next and previous if you prefer.